### PR TITLE
Send Slack notification if propagate-pr fails 

### PR DESCRIPTION
**Pull request by Naatan**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1113" title="DX-1113" target="_blank">StoryDX-1113</a>  There's a clear call to action when version PR merges fail
  </td></table>
  <br />

### CODE REVIEW

File: .github/workflows/propagate.yml [hunk 1]
```diff
@@ -55,5 +55,35 @@ jobs:
...
         name: Propagate PR
+        id: propagate_pr
         shell: bash
-        run: go run scripts/ci/propagate-pr/main.go ${{ github.event.pull_request.number }}
\ No newline at end of file
+        run: go run scripts/ci/propagate-pr/main.go ${{ github.event.pull_request.number }}
+        continue-on-error: true
+
+      - # === Notify Slack of Failures ===
+        name: Notify Slack of Nightly Integration Test Failures
```
Comment (mitchell-as) [inline .github/workflows/propagate.yml:64]: This name is incorrect.

**Review by mitchell-as — CHANGES_REQUESTED** [review]
Looks good other than a name typo.

**Review by mitchell-as — DISMISSED** [review]

**Comment by Naatan** [comment]
@mitchell-as added some slight scope to this story; noticed that failing automation on merge commits are hard to trace back to the PR that caused the failure, so added the PR ID to the merge commit message. Could've been its own story but felt small and relevant enough to include.

**Review by mitchell-as — CHANGES_REQUESTED** [review]

File: scripts/internal/workflow-helpers/github_test.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func TestParseJiraKey(t *testing.T) {
 			actual, err := ParseJiraKey(tc.msg)
 			if err != nil {
 				if tc.expected != "" {
-					t.Errorf("expected %s, got error: %v", tc.expected, err)
+					t.Errorf("expected %s, got error: %v", tc.expected, errs.JoinMessage(err))
```
[the remark this answers is not in the document]
Comment (Naatan) [reply scripts/internal/workflow-helpers/github_test.go:50]: Yep! Technically `%v` will work, but I was not trying to be lazy.. :p

**Review by mitchell-as — DISMISSED** [review]

**Pull request merged**